examples/README.md

```diff
@@ -1,11 +1,50 @@
-## PyTorch distributed example
+## PyTorch distributed examples
 
-Here is an example of something we would aim to setup with the operator. It is just a simple example of distributed pytorch on kubernetes.
-```
-kubectl apply -f multinode/
-```
+Here are examples of jobs we would aim to run with the operator.
 
-The configmap used in the example was created using the distributed training script found in this directory:
-```
-kubectl create configmap dist-train --from-file=dist_train.py
-```
+1. A simple example of distributed MNIST with pytorch on kubernetes:
```

Comment:
@mhbuehler  The first example instructions I had were the plan for the operator. It doesn't actually use the operator. Can we remove instructions on this first example or have a readme for it in it's mnist directory? That or we can remove all the files from mnist aside from the configmap and pytorchjob